packages/utils/src/compat/cache.ts

```diff
@@ -0,0 +1,34 @@
+type Value = string[] | null;
+
+const cache = new Map<string, Map<string[], Value>>();
+
+const _hasCachedValue = (key: string, browsers: string[]): boolean => {
+    return cache.has(key) && cache.get(key)!.has(browsers);
+};
+
+const _getCachedValue = (key: string, browsers: string[]): Value => {
+    return cache.has(key) && cache.get(key)!.get(browsers) || null;
```

Comment:
Nit: `_getCachedValue` gets call always after `_hasCachedValue` so we could potentially drop `cache.has(key)` from this check.

Probably a micro optimization so I'm fine leaving as it is.

Reply:
Yeah, I'd prefer to leave this for now.